pkg/operation/seed/seed.go

```diff
@@ -743,119 +742,6 @@ func RunReconcileSeedFlow(
-			Image:           igwImage.String(),
-			IstiodNamespace: common.IstioNamespace,
-			Annotations:     seed.LoadBalancerServiceAnnotations,
-			Ports:           []corev1.ServicePort{},
-			LoadBalancerIP:  conf.SNI.Ingress.ServiceExternalIP,
-			Labels:          conf.SNI.Ingress.Labels,
-		}
-
-		// even if SNI is being disabled, the existing ports must stay the same
-		// until all APIServer SNI resources are removed.
-		if sniEnabledOrInUse {
-			defaultIngressGatewayConfig.Ports = append(
-				defaultIngressGatewayConfig.Ports,
-				corev1.ServicePort{Name: "proxy", Port: 8443, TargetPort: intstr.FromInt(8443)},
-				corev1.ServicePort{Name: "tcp", Port: 443, TargetPort: intstr.FromInt(9443)},
-				corev1.ServicePort{Name: "tls-tunnel", Port: vpnseedserver.GatewayPort, TargetPort: intstr.FromInt(vpnseedserver.GatewayPort)},
-			)
-		}
-
-		istioDeployers = append(istioDeployers, istio.NewIngressGateway(
-			defaultIngressGatewayConfig,
-			*conf.SNI.Ingress.Namespace,
-			chartApplier,
-			charts.Path,
-			seedClient,
-		))
-
-		// Add for each ExposureClass handler in the config an own Ingress Gateway.
-		for _, handler := range conf.ExposureClassHandlers {
-			istioDeployers = append(istioDeployers, istio.NewIngressGateway(
-				&istio.IngressValues{
-					TrustDomain:     gardencorev1beta1.DefaultDomain,
-					Image:           igwImage.String(),
-					IstiodNamespace: common.IstioNamespace,
-					Annotations:     utils.MergeStringMaps(seed.LoadBalancerServiceAnnotations, handler.LoadBalancerService.Annotations),
-					Ports:           defaultIngressGatewayConfig.Ports,
-					LoadBalancerIP:  handler.SNI.Ingress.ServiceExternalIP,
-					Labels:          gutil.GetMandatoryExposureClassHandlerSNILabels(handler.SNI.Ingress.Labels, handler.Name),
-				},
-				*handler.SNI.Ingress.Namespace,
-				chartApplier,
-				charts.Path,
-				seedClient,
-			))
-		}
-
-		if err := component.OpWaiter(istioDeployers...).Deploy(ctx); err != nil {
-			return err
-		}
-	}
-
-	var proxyGatewayDeployers = []component.DeployWaiter{
-		istio.NewProxyProtocolGateway(
-			&istio.ProxyValues{
-				Labels: conf.SNI.Ingress.Labels,
-			},
-			*conf.SNI.Ingress.Namespace,
-			chartApplier,
-			charts.Path,
-		),
-	}
-
-	for _, handler := range conf.ExposureClassHandlers {
-		proxyGatewayDeployers = append(proxyGatewayDeployers, istio.NewProxyProtocolGateway(
-			&istio.ProxyValues{
-				Labels: gutil.GetMandatoryExposureClassHandlerSNILabels(handler.SNI.Ingress.Labels, handler.Name),
-			},
-			*handler.SNI.Ingress.Namespace,
-			chartApplier,
-			charts.Path,
-		))
-	}
-
-	if gardenletfeatures.FeatureGate.Enabled(features.APIServerSNI) {
-		for _, proxyDeployer := range proxyGatewayDeployers {
-			if err := proxyDeployer.Deploy(ctx); err != nil {
-				return err
-			}
-		}
-	} else {
-		for _, proxyDeployer := range proxyGatewayDeployers {
-			if err := proxyDeployer.Destroy(ctx); err != nil {
-				return err
-			}
-		}
-	}
```

Comment:
Maybe I missed it, but where did this code go? 👀

Reply:
It is in component.go